phoenix-core/src/test/java/org/apache/phoenix/jdbc/CreateTableCompilerTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.phoenix.jdbc;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.apache.phoenix.compile.MutationPlan;
+import org.apache.phoenix.execute.MutationState;
+import org.apache.phoenix.parse.CreateTableStatement;
+import org.apache.phoenix.query.BaseConnectionlessQueryTest;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class CreateTableCompilerTest extends BaseConnectionlessQueryTest {
+
+  @Test
+  public void testCreateTableWithNoVerify() throws SQLException {
+    String ddl = "CREATE TABLE A (K VARCHAR PRIMARY KEY DESC) NOVERIFY";
+    try (Connection conn = DriverManager.getConnection(getUrl());
+         PhoenixStatement phoenixStatement = conn.createStatement().unwrap(PhoenixStatement.class)) {
+      /* Test is placed to 'org.apache.phoenix.jdbc' package to be able to access 'CompilableStatement'
```

Comment:
Can we move this to org.apache.phoenix.compile.PostIndexDDLCompilerTest ?
Based on that test, we don't actually need to acsess CompilableStatement.

Reply:
I need to get the `CompilableStatement` instance to cast it to `CreateTableStatement` and check if the `noverify` is set. 
If we want to check for the `noverify` value in the `Statement` object we need to keep this test here, or if we want to move it we can drop the `assertTrue(((CreateTableStatement) compilableStatement).isNoVerify());` assertion.

Reply:
I'm not a fan of adding a new test class just for this feature.

If the point here is testing that the parser adds a flag, it could be done like in
org.apache.phoenix.parse.QueryParserTest.testParseCreateTableInlinePrimaryKeyWithOrder()
, and add the new test there.

Reply:
Yes, I agree about not introducing new test class. I have separated the tests to check the statement and the mutation in different tests and different test classes, so I could delete this test class without compromising on what we test.